### PR TITLE
refactor: update auth pages to new design

### LINKED ISSUE #114: Update login/signup page
## Description

After getting `@rotki/ui-library@0.1.0` ready we need to update the login/account creation page for the website.

## Tasks

- [ ] TBD

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #114

### CODE REVIEW

**Comment by lukicenturi** [comment]
im adjusting the test

**Comment by lukicenturi** [comment]
will push again after #119 merged

File: components/account/signup/SignupForm.vue [hunk 1]
```diff
@@ -1,420 +1,197 @@
...
+  nextTick(() => {
+    const elem = get(horizontalStepperRef);
+    if (elem) {
+      const a = elem.querySelector('div[class*=active]');
+      a?.scrollIntoView?.({
+        behavior: 'smooth',
+        inline: 'center',
+      });
+    }
+  });
```
Comment (kelsos) [inline components/account/signup/SignupForm.vue:115]: Is this something that would make more sense as part of the Stepper component?

**Review by kelsos — APPROVED** [review]
lgtm I will merge as soon as ci passes

**Pull request merged**